source/03-components/view/views-view/views-view.stories.jsx

```diff
@@ -3,26 +3,24 @@ import parse from 'html-react-parser';
 import { withGlobalWrapper } from '../../../../.storybook/decorators';
 import twigTemplate from './views-view.twig';
 import data from './views-view.yml';
-import pagerTemplate from '../../pager/pager.twig';
-import pagerData from '../../pager/pager.yml';
-import globalData from '../../../00-config/storybook.global-data.yml';
+import ReactDOMServer from 'react-dom/server';
+import { Default as Pager } from '../../pager/pager.stories';
```

Comment:
We typically import external libraries at the top of the file, rather than here in the local stuff. So more like this:
```jsx
import ReactDOMServer from 'react-dom/server';
import parse from 'html-react-parser';

import { withGlobalWrapper } from '../../../../.storybook/decorators';
import twigTemplate from './views-view.twig';
import data from './views-view.yml';
import { Default as Pager } from '../../pager/pager.stories';
```

Reply:
@dcmouyard I don't mind fixing this file, but if we want this to be part of our coding standards, we should make sure it's documented and/or included in eslint.